### ISSUE TITLE
not work on unity 2019.2.9

### ISSUE THREAD

**Issue by toptunes** [gsssrao/UnityAndroidSpeechRecognition#7]
not work on unity 2019.2.9

**Comment by monire91** [comment]
I have same problem i think its not supposed to work on editor u need to test it on phone

**Comment by EricBatlle** [comment]
@monire91 that's right, normally plugins for android involves ADB for debuging, so yes, you need to plug your phone.
